ocaml/sdk-gen/csharp/autogen/src/JsonRpc.cs

```diff
@@ -313,6 +313,8 @@ protected virtual void PerformPostRequest(Stream postStream, Stream responseStre
                         ResponseHeaders.Add(key, string.Join(",", webResponse.Headers.Get(key)));
```

Comment:
This should be: `newResponseHeaders.Add(key, string.Join(",", webResponse.Headers.Get(key)));`